sdk/bpf/c/inc/solana_sdk.h

```diff
@@ -1,741 +1,20 @@
-    int account_infos_len
-) {
-  const SolSignerSeeds signers_seeds[] = {{}};
-  return sol_invoke_signed(
-    instruction,
-    account_infos,
-    account_infos_len,
-    signers_seeds,
-    0
-  );
-}
-
-/**@}*/
-
-/**
- * Debugging utilities
- * @{
- */
-
-/**
- * Prints the hexadecimal representation of a public key
- *
- * @param key The public key to print
- */
-void sol_log_pubkey(
-    const SolPubkey *pubkey
-);
-
-/**
- * Prints the hexadecimal representation of an array
- *
- * @param array The array to print
- */
-static void sol_log_array(const uint8_t *array, int len) {
-  for (int j = 0; j < len; j++) {
-    sol_log_64(0, 0, 0, j, array[j]);
-  }
-}
-
-/**
- * Prints the program's input parameters
- *
- * @param params Pointer to a SolParameters structure
- */
-static void sol_log_params(const SolParameters *params) {
-  sol_log("- Program identifier:");
-  sol_log_pubkey(params->program_id);
-
-  sol_log("- Number of KeyedAccounts");
-  sol_log_64(0, 0, 0, 0, params->ka_num);
-  for (int i = 0; i < params->ka_num; i++) {
-    sol_log("  - Is signer");
-    sol_log_64(0, 0, 0, 0, params->ka[i].is_signer);
-    sol_log("  - Is writable");
-    sol_log_64(0, 0, 0, 0, params->ka[i].is_writable);
-    sol_log("  - Key");
-    sol_log_pubkey(params->ka[i].key);
-    sol_log("  - Lamports");
-    sol_log_64(0, 0, 0, 0, *params->ka[i].lamports);
-    sol_log("  - data");
-    sol_log_array(params->ka[i].data, params->ka[i].data_len);
-    sol_log("  - Owner");
-    sol_log_pubkey(params->ka[i].owner);
-    sol_log("  - Executable");
-    sol_log_64(0, 0, 0, 0, params->ka[i].executable);
-    sol_log("  - Rent Epoch");
-    sol_log_64(0, 0, 0, 0, params->ka[i].rent_epoch);
-  }
-  sol_log("- Instruction data\0");
-  sol_log_array(params->data, params->data_len);
-}
-
-/**@}*/
-
-/**
- * Program instruction entrypoint
- *
- * @param input Buffer of serialized input parameters.  Use sol_deserialize() to decode
- * @return 0 if the instruction executed successfully
- */
-uint64_t entrypoint(const uint8_t *input);
-
-#ifdef SOL_TEST
-/**
- * Stub functions when building tests
- */
-#include <stdio.h>
-void sol_log_(const char *s, uint64_t len) {
-  printf("Program log: %s\n", s);
-}
-void sol_log_64(uint64_t arg1, uint64_t arg2, uint64_t arg3, uint64_t arg4, uint64_t arg5) {
-  printf("Program log: %llu, %llu, %llu, %llu, %llu\n", arg1, arg2, arg3, arg4, arg5);
-}
-void sol_log_pubkey(const SolPubkey *pubkey) {
-  printf("Program log: ");
-  for (int i = 0; i < SIZE_PUBKEY; i++) {
-    printf("%02 ", pubkey->x[i]);
-  }
-  printf("\n");
-}
-void sol_log_compute_units_() {
-  printf("Program consumption: __ units remaining\n");
-}
-void sol_panic_(const char *file, uint64_t len, uint64_t line, uint64_t column) {
-  printf("Panic in %s at %d:%d\n", file, line, column);
-  abort();
-}
-#endif
-
-#ifdef __cplusplus
-}
-#endif
+ * @brief Solana C-based BPF program types utility functions
+ */
+
+#include <deserialize_deprecated.h>
```

Comment:
Should this be `#include <sol_deserialize_deprecated.h>`?  If not the usage of `sol_deserialize_deprecated.h` elsewhere in this PR is probably wrong

Reply:
oops, nm

Reply:
Yeah, it should be